### PR TITLE
Cxx11/chrono lits

### PR DESCRIPTION
**Pull request by mbr0wn**

### CODE REVIEW

**Comment by mbr0wn** [comment]
coveralls tells me that async_writer.h lost some coverage, but I don't understand why or how.

**Comment by sztomi** [comment]
As I look through it, I don't see any difference, so don't worry about it.

File: CMakeLists.txt [hunk 1]
```diff
@@ -30,6 +31,8 @@ set(RPCLIB_VERSION_MAJOR 1)
 set(RPCLIB_VERSION_MINOR 0)
 set(RPCLIB_VERSION_PATCH 0)
 
+add_definitions(-DRPCLIB_DEFAULT_PORT=${RPCLIB_DEFAULT_PORT})
```
Comment (sztomi) [inline CMakeLists.txt]: I'd prefer `target_add_definitions` (you can add it here: https://github.com/rpclib/rpclib/blob/dev/CMakeLists.txt#L135)

**Comment by mbr0wn** [comment]
Rebased onto dev. Moved my defs into your def thing to use `target_compile_definitions`.

**Comment by mbr0wn** [comment]
...and, magically, test coverage went up again :+1:

**Comment by sztomi** [comment]
Perfect, thank you very much.

**Comment by mbr0wn** [comment]
I have more incoming :)

**Pull request merged**